src/frontend-samples/fire-decoration-sample/FireDecorator.ts

```diff
@@ -13,7 +13,7 @@ function randomInteger(min: number, max: number): number {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
 
-/** Generate random floating-point number in [min, max). */
+/** Generate random floating-point number in [min, max]. */
```

Comment:
This was expressed as `[min, max)` because the range includes min but exclude max. Did you change that behavior?

Reply:
Was unaware of this notation.  Reverting change.